FALSEHOODS.md

```diff
@@ -226,6 +226,17 @@ surprising how many false assumptions programmers continue to make about them.
     information. Unless a piece of user-supplied data has actually been verified
     to be a phone number, it should be stored as-is as entered by the user.
 
+1.  **Phone numbers are numbers**
+
+    Never try to store phone numbers as an int or any other kind of numeric
+    data type. You can't do arithmetic on them, and while 007, 07 and 7 are the
+    same number they are not necessarily the same phone number - in some
+    countries a leading 0 is significant and forms part of the number itself
+    (see A leading zero in numbers formatted for domestic usage can always be
+    discarded when dialing from abroad). Moreover, a phone number may contain
```

Comment:
The other one isn't really needed, since it's actually a hyperlink (the xkcd one); but I don't care so to end the back and forth I will just accept this :)